app/ante/ante.go

```diff
@@ -117,6 +117,7 @@ func newCosmosAnteHandler(options cosmosHandlerOptions) sdk.AnteHandler {
 	decorators := []sdk.AnteDecorator{}
 
 	decorators = append(decorators,
+		NewHardLiquidateSpamFilter(1000000),
```

Comment:
can we read this value from config file /environment variable?

Reply:
moved to a constant

Reply:
may not be needed for this specific change, but my thought was that if we wanted to change this limit without having to upgrade the kava binary, reading this value from config or the environment (with a fallback to this default value) would allow us to do that versus hard coding it

Reply:
ya is a good call but going to leave as an improvement since we won't apply this generally to mainnet releases atm.  I need to do more research on gas costs before we tag for all validators